scripts/cluster-upgrade.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+OCP_VERSION_FROM=$1
+OCP_VERSION_TO=$2
+
+if [ -z $OCP_VERSION_FROM ];
+then
+    echo "This script can be used to get the status of all managed clusters or to upgrade clusters."
+    echo "usage: $0 status"
+    echo "usage: $0 <from> <to>"
+    echo "  example: $0 4.1.0 4.1.2"
+    exit 1
+fi
+
+if [ "$OCP_VERSION_FROM" == "status" ];
+then
+    # absense of versions implies status check
+    unset OCP_VERSION_FROM
+    unset OCP_VERSION_TO
+fi
+
+UPGRADE_STARTED=""
+UPGRADE_PROGRESSING=""
+UPGRADE_NOT_POSSIBLE=""
+UPGRADE_DONE=""
+
+NOW_EPOCH=$(date +"%s")
+
+if [ ! -z $OCP_VERSION_FROM ];
```

Comment:
```suggestion
if [ -n $OCP_VERSION_FROM ];
```
The opposite of `! -z` is `-n`, however, I don't think this code will ever be false because earlier you ensure that `$OCP_VERSION_FROM` is set. Thus, this will never be empty.

Reply:
There's a check now that will unset this var if the value starts with "status".